client/src/app/components/stardust/address/section/voting/VotingEvent.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../../../../../scss/fonts";
+@import "../../../../../../scss/mixins";
+@import "../../../../../../scss/media-queries";
+@import "../../../../../../scss/variables";
+@import "../../../../../../scss/themes";
+
+.card.voting-event {
+    .card--header {
+        flex-direction: column;
+        align-items: start;
+        border-bottom: 1px solid var(--border-color);
+    }
+
+    .card--content {
+        .field {
+            border-bottom: 1px solid var(--border-color);
+
+            &:last-child {
+                border: none;
+            }
+        }
+    }
+ 
+    .card--header-info {
+        white-space: pre-wrap;
+        @include font-size(12px);
+
+        color: $gray-5;
+        font-family: $inter;
+        font-weight: 500;
+        letter-spacing: 0.5px;
+
+        @include phone-down {
+            margin-left: 8px;
+            white-space: normal;
+        }
+    }
+
+    .voting-event__status {
+        @include font-size(12px);
+
+        display: flex;
+        align-items: center;
+        height: 24px;
+        margin-right: 8px;
+        padding: 0 8px;
+        border: 0;
+        border-radius: 6px;
+        outline: none;
+        background-color: $gray-light;
+        color: $gray-midnight;
+        font-family: $inter;
+        font-weight: 500;
+        letter-spacing: 0.5px;
+        white-space: nowrap;
+        text-transform: capitalize;
+
+        @include phone-down {
+            height: 32px;
+        }
+        &.voting-event__status--commencing {
+            background-color: var(--message-confirmed-bg);
+            color: $mint-green-7;
+        }
+
+        .voting-event__status--holding {
```

Comment:
This one doesn't have the `&` like the other two sibling classes. Is it on purpose ?